Sofascore2025/EventView.swift

```diff
@@ -0,0 +1,290 @@
+			current.height.equalTo(40)
+			current.width.equalTo(1)
+		}
+		
+//		homeTeamImage constraints
+		homeTeamImage.snp.makeConstraints { current in
+			current.leading.equalToSuperview().inset(80)
+			current.top.equalToSuperview().inset(10)
+			current.width.height.equalTo(16)
+		}
+		
+//		awayTeamImage constraints
+		awayTeamImage.snp.makeConstraints { current in
+			current.leading.equalToSuperview().inset(80)
+			current.top.equalToSuperview().inset(30)
+			current.width.height.equalTo(16)
+		}
+			
+//		homeTeamName constraints
+		homeTeamName.snp.makeConstraints { current in
+			current.leading.equalToSuperview().inset(104)
+			current.centerY.equalTo(homeTeamImage.snp.centerY)
+			current.height.equalTo(16)
+			current.width.equalTo(192)
+		}
+		
+//		awayTeamName constraints
+		awayTeamName.snp.makeConstraints { current in
+			current.leading.equalToSuperview().inset(104)
+			current.centerY.equalTo(awayTeamImage.snp.centerY)
+			current.height.equalTo(16)
+			current.width.equalTo(192)
+		}
+		
+//		homeTeamScore constraints
+		homeTeamScore.snp.makeConstraints { current in
+			current.trailing.equalToSuperview().inset(16)
+			current.centerY.equalTo(homeTeamName.snp.centerY)
+			current.height.equalTo(16)
+			current.width.equalTo(32)
+		}
+		
+//		awayTeamScore constraints
+		awayTeamScore.snp.makeConstraints { current in
+			current.trailing.equalToSuperview().inset(16)
+			current.centerY.equalTo(awayTeamName.snp.centerY)
+			current.height.equalTo(16)
+			current.width.equalTo(32)
+		}
+		
+		
+	}
+	
+	private func setupUpcomingStyle() {
+		homeTeamScore.isHidden = true
+		awayTeamScore.isHidden = true
+		
+//		clock styling
+		clock.textColor = .gray
+		clock.text = "-"
+		
+//		homeTeamName styling
+		homeTeamName.textColor = .black
+		
+//		awayTeamName styling
+		awayTeamName.textColor = .black
+	}
+	
+	private func setupInProgressStyle() {
+		homeTeamScore.isHidden = false
+		awayTeamScore.isHidden = false
+		
+//		clock styling
+		clock.textColor = .red
+		
+//		homeTeamName styling
+		homeTeamName.textColor = .black
+		
+//		awayTeamName styling
+		awayTeamName.textColor = .black
+		
+//		homeTeamScore styling
+		homeTeamScore.textColor = .red
+
+//		awayTeamScore styling
+		awayTeamScore.textColor = .red
+	}
+	
+	private func setupHalftimeStyle() {
+		homeTeamScore.isHidden = false
+		awayTeamScore.isHidden = false
+		
+//		clock styling
+		clock.textColor = .red
+		clock.text = "HT"
+		
+//		homeTeamName styling
+		homeTeamName.textColor = .black
+		
+//		awayTeamName styling
+		awayTeamName.textColor = .black
+		
+//		homeTeamScore styling
+		homeTeamScore.textColor = .red
+		
+//		awayTeamScore styling
+		awayTeamScore.textColor = .red
+	}
+	
+	private func setupFinishedStyle() {
+		homeTeamScore.isHidden = false
+		awayTeamScore.isHidden = false
+		
+//		clock styling
+		clock.textColor = .gray
+		clock.text = "FT"
+	
+//		teamName && teamScore styling
+		if (_match.homeScore! > _match.awayScore!) {
```

Comment:
nikako ne koristimo force unwrapping, tako da pliz makni i ovdje mi se cini da ovo mozes pojednostaviti, npr. samo izracunati koju boju trebas i onda tek na kraju nju samo postavis ovak nesto

```suggestion
    let homeScore = _match.homeScore ?? 0 
    let awayScore = _match.awayScore ?? 0

    let homeTeamColor: UIColor = homeScore > awayScore ? .black : .gray
    let awayTeamColor: UIColor = awayScore > homeScore ? .black : .gray
```